frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./NavBar.css"
+import logo from "../assets/images/logo.png"
+
+const NavBar = ({ nav_1, nav_2, nav_3, nav_4, nav_5 }) => {
+
+   return (
+      <div className="nav">
```

Comment:
The navigation could use a `nav` tag instead of a `div`.

Reply:
Hi Enzo, I made also this two correction in my new commit